CARDS-2323: Embedded view of subject charts and forms, without navigation elements

sdumitriu opened the pull request:
To test:
- start in proms mode
- create a new Visit Information; if the data forms are not automatically created, also create some data forms
- fill in the data forms
- open the visit subject chart
- in the URL, replace `/content.html` with `/embedded.html`, e.g. `http://localhost:8080/embedded.html/Subjects/a10119c8-a9a7-4950-ba7c-48d5ca434dbf/da0201a6-680a-4a5d-8a3f-d59a880e5ae4`
- create a new user, log in with that user and open the embedded view -> error displayed
- as admin, add the user to the Trusted Users group
- as the new user, open again the embedded view -> this time it should display
- repeat these steps for prems mode (except for the Trusted Users part, since prems doesn't have trusted users)

veronikaslc committed:
CARDS-2323: Embedded view of subject charts and forms, without navigation elements

sdumitriu committed:
CARDS-2323: Embedded view of subject charts and forms, without navigation elements
- The resource displayer wasn't loaded
- Fix error in the resource displayer about not being able to load the session attributes (copy-paste leftover)

sdumitriu committed:
CARDS-2323: Embedded view of subject charts and forms, without navigation elements
- Wrong resource name used
- Sometimes the embedded homepage has the wrong type
- Change the endpoint name case from Embedded to embedded

sdumitriu committed:
CARDS-2331: Don't show the close button on the Print Preview if there is no callback to actually handle it

marta- commented on modules/embedded-homepage/src/main/frontend/webpack.config.js:
```diff
@@ -0,0 +1,34 @@
+module.exports = {
+  mode: 'development',
+  entry: {
+    [module_name + 'themeindex']: './src/embeddedThemePage/index.jsx'
```
Not a big deal, but why are we calling it "themeindex" instead of just "index"?

sdumitriu replied:
Done.

sdumitriu committed:
CARDS-2323: Embedded view of subject charts and forms, without navigation elements
- Rename the module to embedded-view instead of embedded-homepage
- Rename the JSX entry point from themeindex.js to index.js
- Enable the embedded view globally, not just in proms
- Fix copy-pasted details in the frontend package description

marta- reviewed (approved):
Reviewed and tested in proms, everything works as expected. I might want to polish the styling a bit, but it doesn't have to hold the merge.

marta- commented:
> * create a new Visit Information; if the data forms are not automatically created, also create some data forms

I'm not encountering any issues with data form creations. FTR during tests my visits were generally scheduled ~3 days from now to mimic the proms use case.

sdumitriu reviewed (commented):
Approving (cannot self-approve the PR), everything seems to work fine.

sashaandjic commented:
Proms mode:
Trusted user:
![image](https://github.com/data-team-uhn/cards/assets/23243244/b914dac8-b818-48e6-b926-eab06ee558ae)

User not in the trusted group:
![image](https://github.com/data-team-uhn/cards/assets/23243244/52ced2c4-24f2-4227-90df-37d96859117b)

sashaandjic commented:
I get the same behavior in prems. Only trusted users can access forms.
